.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Help us to improve PatternFly
+title: Bug - [Component] - [short description]
+labels: bug, needs triage
+assignees: ''
+
+---
+
+**Describe the problem**
+A clear and concise description of the problem. Which components are affected?
+
+**How do you reproduce the problem?**
+Provide steps to reproduce. A [codepen](http://codepen.io) or [codesandbox](https://codesandbox.io/) demonstrating the problem is appreciated.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Is this issue blocking you?**
+List the workaround if there is one.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**What is your environment?**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**What is your product and what release version are you targeting?**
```

Comment:
Maybe release date? To me it seems more meaningful than a version, unless they also mention the date with the version.

```suggestion
**What is your product and what release date are you targeting?**
```